app/frontend/tests/unit/views/RequestAccount.spec.js

```diff
@@ -11,9 +11,9 @@ describe('Secure.vue', () => {
   });
 
   it('renders', () => {
-    const wrapper = shallowMount(Secure, {
+    const wrapper = shallowMount(RequestAccount, {
       vuetify,
-      stubs: ['BaseSecure']
+      stubs: ['BaseSecure', 'RequestForm']
     });
 
     expect(wrapper.text()).toMatch('');
```

Comment:
not much text to look for. 
I guess we need a separate test for the child RequestForm component

Reply:
Yes - RequestForm needs its own test in its entirety as there's some logic in there that needs to be covered for. That's out of scope of this specific PR though as we have to sort out how we want to abstract out the keycloak specific pieces of information in order to make it mockable properly.